src/hooks/useAppState/useAppState.test.js

```diff
@@ -93,7 +118,7 @@ describe('useAppState', () => {
     // First time a list is created
     await waitFor(() => {
       expect(mockFetch).toHaveBeenCalledWith(...createListArgs);
-      expect(mockFetch).toHaveBeenCalledWith(...getItemsArgs);
+      expect(mockFetch).not.toHaveBeenCalledWith(...getItemsArgs);
```

Comment:
This was a bit weird before, we used to create a list then immediately check if there was something in the list (which there never was), we no longer do that